frontend/src/metabase/status/components/FileUploadStatusLarge/FileUploadStatusLarge.tsx

```diff
@@ -90,9 +90,15 @@ const getTitle = (
   uploadDestination: Collection | Table,
 ) => {
   const isDone = uploads.every(isUploadCompleted);
+  const isOnlyReplace = uploads.every(
+    upload => upload.uploadMode === "replace",
+  );
   const isError = uploads.some(isUploadAborted);
 
   if (isDone) {
+    if (isOnlyReplace) {
```

Comment:
Moot, as I don't believe we support multi-uploads currently, and even then wouldn't support mixing verbs, but I'd expect this to be the message if *any* of the uploads were doing a replacement.

Reply:
As far as the frontend is concerned, you can have multiple pending uploads, they're separate requests as far as the backend is concerned.

I'll run the copy by product - the UI here is a little tricky since there can be one "title" for multiple uploads doing slightly different things.

Reply:
Ah, I'd guess incorrectly that the plural related to multiple files being uploaded at once.

Reply:
well, they can be uploaded at once, just in separate pending requests

Reply:
I mean as in a multi-file picker appending multiple CSVs in a single "upload" action.